src/Riok.Mapperly/Descriptors/MappingBodyBuilder/NewInstanceObjectPropertyMappingBodyBuilder.cs

```diff
@@ -195,42 +195,88 @@ private static void BuildInitOnlyPropertyMappings(NewInstanceMappingBuilderConte
     {
         constructorParameterMappings = new HashSet<ConstructorParameterMapping>();
         mappedTargetPropertyNames = new HashSet<string>();
-        var skippedOptionalParam = false;
+        bool skippedOptionalParam = false;
```

Comment:
I would stick with `var` as it was used before and it is used all across the codebase.